cardano-config/src/Cardano/Config/TraceConfig.hs

```diff
@@ -109,6 +110,7 @@ data TraceConfig'
   , traceDnsSubscription :: !Bool
   , traceErrorPolicy :: !Bool
   , traceForge :: !Bool
+  , traceForgeState :: !Bool
```

Comment:
FYI, we trace the forge state every slot. So for Byron once every 20s and for Shelley once every 1 or 2s.